vm/src/stdlib/posix.rs

```diff
@@ -39,8 +39,8 @@ pub mod module {
             _os, fs_metadata,
         },
         utils::{Either, ToCString},
-        IntoPyObject, ItemProtocol, PyObjectRef, PyResult, PyValue, StaticType, TryFromObject,
-        VirtualMachine,
+        IntoPyObject, ItemProtocol, PyObjectRef, PyRef, PyResult, PyValue, StaticType,
```

Comment:
@zetwhite could you check these unused import warnings?

Reply:
Ok. i am going to check it